pkg/blob/local/localstorage.go

```diff
@@ -47,8 +47,11 @@ func InitLocalStorage() error {
 			log.Errorf("InitLocalStorage: Error reading SegSetKeys file %s: %v", filePath, err)
 			return err
 		}
+	} else if len(file) == 0 {
```

Comment:
thsi code could be formatted a bit better:
`if err != nil {
		... blah blha
	} 

if len(file) == 0 {
 log error and return
}
err := json.Unmarshal(file, &segSetKeys)
.....
`
this way it reduces the deep if-els-eif-else...